bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/DownloadServlet.java

```diff
@@ -15,6 +15,8 @@
  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
 package com.adobe.cq.wcm.core.components.internal.servlets;
 
+import static com.adobe.cq.wcm.core.components.internal.servlets.DownloadServlet.SELECTOR;
```

Comment:
why is this needed?

Reply:
Was a shortcut for the selector in the `Component` annotation. But right, maybe it's mor clear to remove the the static import. done in https://github.com/adobe/aem-core-wcm-components/pull/724/commits/87e08a6658f8f16930ae17626490848f8f94dd49